src/js/live-tracker.js

```diff
@@ -107,9 +113,13 @@ class LiveTracker extends Component {
    * and start/stop tracking accordingly.
    */
   handleDurationchange() {
-    if (this.player_.duration() === Infinity) {
+    if (this.player_.duration() === Infinity && this.liveWindow() > this.options_.trackingThreshold) {
```

Comment:
The `liveui` is now activated in the `LiveTracker` instead of `Player` so that we can add in things like `trackingThreshold`.

Reply:
Really minor, but we may want to change the wording in the defaults comment or change this to >=.

Reply:
changed to `>=`, good catch.